Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM grubykarol/locust:0.13.5-python3.8-alpine3.11
+
+# Install OpenJDK-8
+RUN apk update
+RUN apk add --no-cache openjdk8
+
+# Setup JAVA_HOME -- useful for docker commandline
+
+ENV JAVA_HOME /usr/lib/jvm/java-1.8-openjdk
+ENV PATH $PATH:/usr/lib/jvm/java-1.8-openjdk/jre/bin:/usr/lib/jvm/java-1.8-openjdk/bin
+
```

Comment:
We need to add the fontconfig, due that is not included in alpine and it generate an error. The following line, needs to be added:
RUN apk --update add fontconfig ttf-dejavu

Reply:
Fixed! Thanks for the explanations!